<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="Project Test Suite" tests="31" assertions="72" errors="0" failures="0" skipped="0" time="36.388043">
    <testsuite name="Tests\AppBundle\Command\CreateAgentCommandTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Command/CreateAgentCommandTest.php" tests="1" assertions="1" errors="0" failures="0" skipped="0" time="23.969133">
      <testcase name="testExecute" class="Tests\AppBundle\Command\CreateAgentCommandTest" classname="Tests.AppBundle.Command.CreateAgentCommandTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Command/CreateAgentCommandTest.php" line="11" assertions="1" time="23.969133"/>
    </testsuite>
    <testsuite name="Tests\AppBundle\Controller\CalculatorControllerTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Controller/CalculatorControllerTest.php" tests="2" assertions="4" errors="0" failures="0" skipped="0" time="11.343064">
      <testcase name="testAdd" class="Tests\AppBundle\Controller\CalculatorControllerTest" classname="Tests.AppBundle.Controller.CalculatorControllerTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Controller/CalculatorControllerTest.php" line="13" assertions="1" time="0.002956"/>
      <testsuite name="Tests\AppBundle\Controller\CalculatorControllerTest::testCalculator" tests="1" assertions="3" errors="0" failures="0" skipped="0" time="11.340108">
        <testcase name="testCalculator with data set #0" class="Tests\AppBundle\Controller\CalculatorControllerTest" classname="Tests.AppBundle.Controller.CalculatorControllerTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Controller/CalculatorControllerTest.php" line="26" assertions="3" time="11.340108"/>
      </testsuite>
    </testsuite>
    <testsuite name="Tests\AppBundle\Controller\DefaultControllerTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Controller/DefaultControllerTest.php" tests="1" assertions="2" errors="0" failures="0" skipped="0" time="0.238442">
      <testcase name="testIndex" class="Tests\AppBundle\Controller\DefaultControllerTest" classname="Tests.AppBundle.Controller.DefaultControllerTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Controller/DefaultControllerTest.php" line="9" assertions="2" time="0.238442"/>
    </testsuite>
    <testsuite name="AppBundle\Tests\EventListener\RequestListenerTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/EventListener/RequestListenerTest.php" tests="1" assertions="1" errors="0" failures="0" skipped="0" time="0.028712">
      <testcase name="testListenerThrowsWhenMasterRequestHasInconsistentClientIps" class="AppBundle\Tests\EventListener\RequestListenerTest" classname="AppBundle.Tests.EventListener.RequestListenerTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/EventListener/RequestListenerTest.php" line="18" assertions="1" time="0.028712"/>
    </testsuite>
    <testsuite name="AppBundle\Tests\EventListener\ResponseListenerTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/EventListener/ResponseListenerTest.php" tests="1" assertions="1" errors="0" failures="0" skipped="0" time="0.006521">
      <testcase name="testFilterDoesNothingForSubRequests" class="AppBundle\Tests\EventListener\ResponseListenerTest" classname="AppBundle.Tests.EventListener.ResponseListenerTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/EventListener/ResponseListenerTest.php" line="31" assertions="1" time="0.006521"/>
    </testsuite>
    <testsuite name="Tests\AppBundle\Repository\CustomerRepositoryTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Repository/CustomerRepositoryTest.php" tests="5" assertions="11" errors="0" failures="0" skipped="0" time="0.542916">
      <testsuite name="Tests\AppBundle\Repository\CustomerRepositoryTest::findAllRecordsTest" tests="3" assertions="9" errors="0" failures="0" skipped="0" time="0.363401">
        <testcase name="findAllRecordsTest with data set #0" class="Tests\AppBundle\Repository\CustomerRepositoryTest" classname="Tests.AppBundle.Repository.CustomerRepositoryTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Repository/CustomerRepositoryTest.php" line="32" assertions="3" time="0.216593"/>
        <testcase name="findAllRecordsTest with data set #1" class="Tests\AppBundle\Repository\CustomerRepositoryTest" classname="Tests.AppBundle.Repository.CustomerRepositoryTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Repository/CustomerRepositoryTest.php" line="32" assertions="3" time="0.073292"/>
        <testcase name="findAllRecordsTest with data set #2" class="Tests\AppBundle\Repository\CustomerRepositoryTest" classname="Tests.AppBundle.Repository.CustomerRepositoryTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Repository/CustomerRepositoryTest.php" line="32" assertions="3" time="0.073516"/>
      </testsuite>
      <testsuite name="Tests\AppBundle\Repository\CustomerRepositoryTest::findOneByIdTest" tests="2" assertions="2" errors="0" failures="0" skipped="0" time="0.179515">
        <testcase name="findOneByIdTest with data set #0" class="Tests\AppBundle\Repository\CustomerRepositoryTest" classname="Tests.AppBundle.Repository.CustomerRepositoryTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Repository/CustomerRepositoryTest.php" line="65" assertions="1" time="0.093966"/>
        <testcase name="findOneByIdTest with data set #1" class="Tests\AppBundle\Repository\CustomerRepositoryTest" classname="Tests.AppBundle.Repository.CustomerRepositoryTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Repository/CustomerRepositoryTest.php" line="65" assertions="1" time="0.085549"/>
      </testsuite>
    </testsuite>
    <testsuite name="tests\PhpunitBundle\Service\CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" tests="10" assertions="26" errors="0" failures="0" skipped="0" time="0.147265">
      <testcase name="testCreateOne" class="tests\PhpunitBundle\Service\CustomerServiceTest" classname="tests.PhpunitBundle.Service.CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" line="44" assertions="4" time="0.065016"/>
      <testsuite name="tests\PhpunitBundle\Service\CustomerServiceTest::testGetAll" tests="3" assertions="6" errors="0" failures="0" skipped="0" time="0.022332">
        <testcase name="testGetAll with data set #0" class="tests\PhpunitBundle\Service\CustomerServiceTest" classname="tests.PhpunitBundle.Service.CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" line="68" assertions="2" time="0.009402"/>
        <testcase name="testGetAll with data set #1" class="tests\PhpunitBundle\Service\CustomerServiceTest" classname="tests.PhpunitBundle.Service.CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" line="68" assertions="2" time="0.006496"/>
        <testcase name="testGetAll with data set #2" class="tests\PhpunitBundle\Service\CustomerServiceTest" classname="tests.PhpunitBundle.Service.CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" line="68" assertions="2" time="0.006434"/>
      </testsuite>
      <testsuite name="tests\PhpunitBundle\Service\CustomerServiceTest::testGetOne" tests="2" assertions="2" errors="0" failures="0" skipped="0" time="0.013962">
        <testcase name="testGetOne with data set #0" class="tests\PhpunitBundle\Service\CustomerServiceTest" classname="tests.PhpunitBundle.Service.CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" line="87" assertions="1" time="0.006968"/>
        <testcase name="testGetOne with data set #1" class="tests\PhpunitBundle\Service\CustomerServiceTest" classname="tests.PhpunitBundle.Service.CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" line="87" assertions="1" time="0.006994"/>
      </testsuite>
      <testcase name="testUpdateOneShouldThrowExceptionForInvalidCustomerId" class="tests\PhpunitBundle\Service\CustomerServiceTest" classname="tests.PhpunitBundle.Service.CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" line="104" assertions="4" time="0.012454"/>
      <testsuite name="tests\PhpunitBundle\Service\CustomerServiceTest::testUpdateOne" tests="1" assertions="4" errors="0" failures="0" skipped="0" time="0.010725">
        <testcase name="testUpdateOne with data set #0" class="tests\PhpunitBundle\Service\CustomerServiceTest" classname="tests.PhpunitBundle.Service.CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" line="128" assertions="4" time="0.010725"/>
      </testsuite>
      <testcase name="testDeleteOneShouldThrowExceptionForInvalidCustomerId" class="tests\PhpunitBundle\Service\CustomerServiceTest" classname="tests.PhpunitBundle.Service.CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" line="151" assertions="4" time="0.007710"/>
      <testsuite name="tests\PhpunitBundle\Service\CustomerServiceTest::testDeleteOne" tests="1" assertions="2" errors="0" failures="0" skipped="0" time="0.015066">
        <testcase name="testDeleteOne with data set #0" class="tests\PhpunitBundle\Service\CustomerServiceTest" classname="tests.PhpunitBundle.Service.CustomerServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/CustomerServiceTest.php" line="175" assertions="2" time="0.015066"/>
      </testsuite>
    </testsuite>
    <testsuite name="tests\PhpunitBundle\Service\ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" tests="10" assertions="26" errors="0" failures="0" skipped="0" time="0.111991">
      <testcase name="testCreateOne" class="tests\PhpunitBundle\Service\ProductServiceTest" classname="tests.PhpunitBundle.Service.ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" line="44" assertions="4" time="0.032184"/>
      <testsuite name="tests\PhpunitBundle\Service\ProductServiceTest::testGetAll" tests="3" assertions="6" errors="0" failures="0" skipped="0" time="0.019844">
        <testcase name="testGetAll with data set #0" class="tests\PhpunitBundle\Service\ProductServiceTest" classname="tests.PhpunitBundle.Service.ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" line="76" assertions="2" time="0.007128"/>
        <testcase name="testGetAll with data set #1" class="tests\PhpunitBundle\Service\ProductServiceTest" classname="tests.PhpunitBundle.Service.ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" line="76" assertions="2" time="0.006492"/>
        <testcase name="testGetAll with data set #2" class="tests\PhpunitBundle\Service\ProductServiceTest" classname="tests.PhpunitBundle.Service.ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" line="76" assertions="2" time="0.006224"/>
      </testsuite>
      <testsuite name="tests\PhpunitBundle\Service\ProductServiceTest::testGetOne" tests="2" assertions="2" errors="0" failures="0" skipped="0" time="0.015211">
        <testcase name="testGetOne with data set #0" class="tests\PhpunitBundle\Service\ProductServiceTest" classname="tests.PhpunitBundle.Service.ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" line="95" assertions="1" time="0.008163"/>
        <testcase name="testGetOne with data set #1" class="tests\PhpunitBundle\Service\ProductServiceTest" classname="tests.PhpunitBundle.Service.ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" line="95" assertions="1" time="0.007048"/>
      </testsuite>
      <testcase name="testUpdateOneShouldThrowExceptionForInvalidProductId" class="tests\PhpunitBundle\Service\ProductServiceTest" classname="tests.PhpunitBundle.Service.ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" line="112" assertions="4" time="0.008006"/>
      <testsuite name="tests\PhpunitBundle\Service\ProductServiceTest::testUpdateOne" tests="1" assertions="4" errors="0" failures="0" skipped="0" time="0.011971">
        <testcase name="testUpdateOne with data set #0" class="tests\PhpunitBundle\Service\ProductServiceTest" classname="tests.PhpunitBundle.Service.ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" line="136" assertions="4" time="0.011971"/>
      </testsuite>
      <testcase name="testDeleteOneShouldThrowExceptionForInvalidProductId" class="tests\PhpunitBundle\Service\ProductServiceTest" classname="tests.PhpunitBundle.Service.ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" line="159" assertions="4" time="0.008968"/>
      <testsuite name="tests\PhpunitBundle\Service\ProductServiceTest::testDeleteOne" tests="1" assertions="2" errors="0" failures="0" skipped="0" time="0.015806">
        <testcase name="testDeleteOne with data set #0" class="tests\PhpunitBundle\Service\ProductServiceTest" classname="tests.PhpunitBundle.Service.ProductServiceTest" file="/var/www/html/TDD/symfony_3.4/tests/AppBundle/Service/ProductServiceTest.php" line="183" assertions="2" time="0.015806"/>
      </testsuite>
    </testsuite>
  </testsuite>
</testsuites>
